{{cookiecutter.project_dashed}}-src/ansible/group_vars/all

```diff
@@ -1,16 +1,18 @@
-
 use_pvt_dns: false
 
 project_ident: {{ cookiecutter.project_namespace }}
+project_ident_env: {{ cookiecutter.project_namespace }}-{{ '{{' }} app_environment {{'}}'}}
 
-app_db_host: "{{ cookiecutter.database_pvt_host }}"
 {% raw %}
+app_db_host: "{{ lookup('dig', '+short', 'sierra.pvt.level12.net', '@ns1.linode.com')}}"
```

Comment:
You have defined database_pvt_host in the cookiecutter variables, but then you don't use it here.